lua/telescope/_extensions/file_browser/make_entry.lua

```diff
@@ -157,6 +159,17 @@ local make_entry = function(opts)
       table.insert(widths, { width = strings.strdisplaywidth(icon) })
       table.insert(display_array, { icon, icon_hl })
     end
+
+    if opts.git_status then
+      if entry.value == parent_dir then
+        table.insert(widths, { width = 1 })
+        table.insert(display_array, { " ", icon_hl })
```

Comment:
I'm reasonably sure off the top of my head 

```lua
        table.insert(display_array, " ")
```

it's just a tuple if there is an associated highlight group.

But yeah, doesn't matter too much.

Reply:
What you're trying to say is that the table isn't required? Sounds great! Should I verify this somewhere first?

Reply:
I tried it out and it works!